app/views/circuitverse/tos.html.erb

```diff
@@ -1,6 +1,10 @@
 <% content_for :title, t("circuitverse.tos.title") %>
 <div class="container">
-  <h1><%= t("circuitverse.tos.main_heading") %></h1>
+  <div class="row center-row">
+    <div class="col-xs col-sm col-md col-lg">
+      <h1 class="mt-4"><%= t("circuitverse.tos.main_heading") %></h1>
+    </div>
+  </div>
```

Comment:
Instead of this you can just use,

```suggestion
  <h1 class="text-center my-4"><%= t("circuitverse.tos.main_heading") %></h1>
```

Same of privacy section